REVIEWING.md

```diff
@@ -0,0 +1,28 @@
+# Reviewing Yari changes
+
+This document provides information on how to review changes to the yari repo.
+
+## Before you start
+
+Set up the yari repo and the corresponding [content repo](https://github.com/mdn/content) locally, as described in the [Yari quickstart](README.md#quickstart) guide. Once you've got them successfully set up, run the `yarn` and `yarn dev` commands to update your fork with the latest packages and start the MDN test site running locally on `localhost:3000`.
+
+`yarn dev` is slower to execute than `yarn start` but it makes sure `client/build/` is clean, and it also reloads the SSR part of the site (it does a fresh Webpack build). This might matter after you've run `git pull` & `yarn`, since certain packages might be upgraded and it's always good to test with a clean, up-to-date build.
+
+Make sure you set the `CONTENT_ROOT` environment variable to an absolute path to the `content` repo `files` sub-directory before running `yarn dev`. This can be done using a command like `export CONTENT_ROOT=/Users/path/to/content/files`.
+
+## General testing procedure
+
+When you are tasked with reviewing a yari pull request:
+
+1. Go to your local fork clone, switch to a new branch for testing, and pull the PR branch into it.
+2. Next, run `yarn` to pull in any package changes, then `yarn dev` to build the site and start the local server.
+3. Now go to the local server to test the change.
+4. Provide feedback. Be helpful, and above all, welcoming and friendly.
+
+## Testing KumaScript macro changes
+
+The legacy [KumaScript](https://developer.mozilla.org/en-US/docs/MDN/Tools/KumaScript) macro system is available inside the yari repo in the [kumascript](https://github.com/mdn/yari/tree/master/kumascript) subdirectory.
```

Comment:
"subdirectory" or "sub-directory"? I honestly don't know, it's not important either. But I noticed you used hyphen version above but not here. So just curious for my own learning.

Reply:
I don't think it really matters, as long as it is consistent. I've gone for "subdirectory"

Reply:
It could make a difference for accessibility reasons (esp. learning disabilities).
I think, Wikipedia calls translation into this subset of English „Plain English” (German's „Leichte Sprache”).